### PR TITLE
feat: adds correct status link + indicator [ch-000]

### PR DESCRIPTION
**Pull request by tnolet**
Adds the correct link to the new status page + a nice indicator

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -276,3 +276,24 @@ $(document).ready(() => {
     }
   })
 })
+
+/**
+ * Fetch status from statuspage.com and update link in footer
+ */
+$(document).ready(() => {
+  fetch('https://nq8lf8mrmvw6.statuspage.io/api/v2/status.json')
```
Comment (maxigimenez) [inline src/js/index.js:284]: BTW this is a nice feature for our status page

**Review by maxigimenez — APPROVED** [review]
LGTM 👍

**Review by ianaya89 — CHANGES_REQUESTED** [review]
Looks good but indicator is not visible

image

**Comment by tnolet** [comment]
> Looks good but indicator is not visible
> 
> 
> 
> image
> 
> 

I see. On mobile it does not show indeed. Good catch

File: src/js/index.js (hunk 1 above)
Comment (tnolet) [reply src/js/index.js:284]: @maxigimenez wadda ya mean?

**Comment by tnolet** [comment]
> > Looks good but indicator is not visible
> > image
> 
> I see. On mobile it does not show indeed. Good catch

@ianaya89 seems like the `dot--green` css class does not exist. That is very weird. This works great on my local dev. Famous last word.

File: src/js/index.js (hunk 1 above)
Comment (maxigimenez) [reply src/js/index.js:284]: expose a json endpoint to easily grab the overall status, so customer can easily add indicators anywhere like we are doing

**Comment by tnolet** [comment]
@maxigimenez @ianaya89 this is sooo weird. Somehow that `dot--green` class does not exist on the Vercel deploy. Works great on my machine.

**Review by ianaya89 — APPROVED** [review]

**Pull request merged**